doc/Language/pod.rakudoc

```diff
@@ -295,54 +543,134 @@ C<=end para> block, any blank lines are preserved.
+As the previous example demonstrates, within an explicit C<=code> block the code can start at the (virtual) left margin.
+Furthermore, lines that start with whitespace characters after that margin have subsequent whitespace preserved exactly
+(in addition to the implicit nesting of the code). Explicit C<=code> blocks may also contain empty lines.
 
-Code blocks can also be explicitly defined by enclosing them in C<=begin code>
-and C<=end code>
+=head3 Formatting within code blocks
 
-=begin code :lang<rakudoc>
-    =begin code
-    my $name = 'John Doe';
-    say $name;
+Although C<=code> blocks automatically disregard all formatting codes, occasionally you may still
+need to specify some formatting within a code block. For example, you may wish to
+emphasize a particular keyword in an example (using a C<B<>> code). Or you may want to indicate that
+part of the example is metasyntactic (using the C<R<>> code). Or you might need to insert a non-ASCII
+character (using the C<E<>> code).
+
+You can specify a list of formatting codes that should still be recognized within a code block using
+the C<:allow> option. The value of the C<:allow> option must be a list of the (single-letter)
+names of one or more formatting codes. Those codes will then remain active inside the code block.
+For example:
+=begin code
+    =begin code :allow< B R >
+    sub demo {
+        B<say> 'Hello R<name>';
+        I<note> 'The I format is not recognised';
+    }
     =end code
 =end code
 
+would be rendered:
+
+=begin code :allow< B R >
+sub demo {
+    B<say> 'Hello R<name>';
+    I<note> 'The I format is not recognised';
+    }
+=end code
+
 =head2 I/O blocks
 
-Pod6 provides blocks for specifying the input and output of programs.
+Rakudoc provides blocks for specifying the input and output of programs.
+
+=for nested
+B<Note> The Rakudo compiler does not (yet) produce C<Pod::Block>s for Input or Output
+and instead produces C<Pod::Block::Code> for C<=code>, C<=input> and C<=output>.
 
 The C<=input> block is used to specify pre-formatted keyboard input,
 which should be rendered without re-justification or squeezing of whitespace.
 
 The C<=output> block is used to specify pre-formatted terminal or file output,
 which should also be rendered without re-justification or whitespace-squeezing.
+=begin code
+
+    =begin output
+            Name:    Baracus, B.A.
+            Rank:    Sgt
+            Serial:  1PTDF007
+
+            Do you want additional personnel details? K<y>
+
+            Height:  180cm/5'11"
+            Weight:  104kg/230lb
+            Age:     49
+
+            Print? K<n>
+        =end output
+
+=end code
 
 =head2 Lists
 
-=head3 Unordered lists
+Lists in Rakudoc are specified as a series of contiguous C<=item> blocks.
+No special "container" directives or other delimiters are required to enclose the entire list.
 
-Lists in Pod6 are specified as a series of C<=item> blocks.
+Note that C<=item> is just an abbreviation for C<=item1>.
```

Comment:
s/just//

Reply:
Sure? Emphasizing that it isn't a first-class thing might even be good.